### pr등록 후 애플리케이션 실행 시 에러발생
- [ERROR] 2024-01-05 17:08:17 [main] [SpringApplication:839] - Application run failed

### 내용
- 에러 내용들 확인&해결 후 다시 올리겠습니다.

### 에러 내용
- Error creating bean with name 'loginController'
- Error creating bean with name 'userRepository'
- No property 'email' found for type 'User'

--- a/src/main/java/org/ccs/app/core/user/infra/repository/UserRepository.java
+++ b/src/main/java/org/ccs/app/core/user/infra/repository/UserRepository.java
@@ -11,6 +11,6 @@ import org.springframework.stereotype.Repository;
 
 // TODO : DB 연결 후 수정
 @Repository
-public interface UserRepository extends JpaRepository<User, Long>, UserCustomRepository {
+public interface UserRepository extends JpaRepository<UserAccount, Long>, UserCustomRepository {
     UserAccount findByEmail(String email);
 }

